aea/protocols/generator/common.py

```diff
@@ -364,7 +364,7 @@ def _create_protocol_file(
     """
     pathname = os.path.join(path_to_protocol_package, file_name)
 
-    with open(pathname, "w") as file:
+    with open(pathname, "w", newline="\n") as file:
```

Comment:
why this? I guess related to #1631

Reply:
the reason, that windows generation differs from unix one cause new lines. so decided to unify it